velox/functions/prestosql/aggregates/tests/ApproxDistinctTest.cpp

```diff
@@ -304,6 +324,16 @@ TEST_F(ApproxDistinctTest, globalAggAllNulls) {
   EXPECT_TRUE(readSingleValue(op).isNull());
 }
 
+TEST_F(ApproxDistinctTest, hugeInt) {
+  auto hugeIntValues =
```

Comment:
Seems like you added testWithTableScan as false because AggregationFuzzer uses DWRF that doesn't support HUGEINT ? Please add a comment about it.